ngx_http_sxg_filter_module.c

```diff
@@ -687,25 +687,32 @@ static ngx_int_t ngx_http_sxg_filter_init(ngx_conf_t* cf) {
     }
 
     if (!is_valid_config(cf, nscf)) {
-      ngx_log_error(NGX_LOG_NOTICE, cf->log, 0,
+      ngx_log_error(NGX_LOG_EMERG, cf->log, 0,
                     "nginx-sxg-module: invalid config");
       return NGX_ERROR;
     }
 
     EVP_PKEY* privkey =
         load_private_key((const char*)nscf->certificate_key.data);
     if (privkey == NULL) {
-      ngx_log_error(NGX_LOG_NOTICE, cf->log, 0,
+      ngx_log_error(NGX_LOG_EMERG, cf->log, 0,
                     "nginx-sxg-module: failed to load private key %V",
                     &nscf->certificate_key);
       return NGX_ERROR;
     }
     X509* cert = load_x509_cert((const char*)nscf->certificate.data);
+    if (cert == NULL) {
+      ngx_log_error(NGX_LOG_EMERG, cf->log, 0,
+                    "nginx-sxg-module: failed to load certificate %V",
+                    &nscf->certificate);
+      return NGX_ERROR;
+    }
+
     if (!sxg_add_ecdsa_signer("nginx", /*date=*/0, /*expires=*/0,
                               (const char*)nscf->validity_url.data, privkey,
                               cert, (const char*)nscf->cert_url.data,
                               &nscf->signers)) {
-      ngx_log_error(NGX_LOG_NOTICE, cf->log, 0,
+      ngx_log_error(NGX_LOG_EMERG, cf->log, 0,
```

Comment:
Can you distinguish this log from the previous one, e.g., Could not load (x509|ecdsa) certificate?

Reply:
Actually, failure in `sxg_add_ecdsa_signer` means failure of `OPENSSL_malloc`. So modified this to `failed to allocate memory`